### DISCUSSION TITLE
Custom `RenderGraph`

### DISCUSSION THREAD

**Discussion by 68317fa2** [bevyengine/bevy#15684] [Q&A]
Are there any examples, tutorials or resources on making a custom `RenderGraph`. More specifically, a `RenderGraph` as a `SubGraph` of the main-`RenderGraph`, but isolated from `core3d` and `core2d` and for custom `Cameras`. I want to write a custom ray-tracer and want to get the whole pipelining and rendering structure done, so I can get going with some compute shaders. Are there any things I can do?

**Comment by eatenpancreas** [comment]
Agreed, it would be nice to see more shader stuff.
